core/storage/storage_models_test.py

```diff
@@ -154,12 +154,31 @@ def test_get_models_which_should_be_exported(self):
         for model in all_models:
             export_policy = model.get_export_policy()
             if model in models_with_export:
-                self.assertEqual(
-                    base_models.EXPORT_POLICY.CONTAINS_USER_DATA,
-                    export_policy
+                self.assertTrue(
+                    base_models.EXPORT_POLICY.EXPORTED in export_policy.values()
                 )
             else:
-                self.assertEqual(
-                    base_models.EXPORT_POLICY.NOT_APPLICABLE,
-                    export_policy
+                self.assertTrue(
+                    base_models.EXPORT_POLICY.EXPORTED
+                    not in export_policy.values()
+                )
+
+    def test_all_fields_have_export_policy(self):
+        """Ensure every field in every model has an export policy defined."""
+        all_models = [
+            clazz
+            for clazz in self._get_model_classes()
+            if not clazz.__name__ in self.BASE_CLASSES
+        ]
+        for model in all_models:
+            export_policy = model.get_export_policy()
+            for field in model._properties: #pylint: disable=protected-access
+                self.assertTrue(
+                    field in export_policy.keys()
+                )
+                self.assertTrue(
```

Comment:
Again, perhaps cleaner to just make a set with all the values, and check that it's a subset of the desired set. Then you don't need a for loop.

Reply:
Done